### ISSUE TITLE
[North Star Applicant UI] Verify aria labels are translated

### ISSUE THREAD

**Issue by greg-hyde** [civiform/civiform#7954]
I think there are some spots where the messages are translated, but not HTML aria labels. We should go through fragments/templates and look for instances of this.  

First, audit in Staging to see if any translations are missing.  If translations are needed, create a separate implementation issue.

**Comment by ssandbekkhaug** [comment]
Here's an example in [NavigationFragment](https://github.com/civiform/civiform/blob/main/server/app/views/applicant/NavigationFragment.html
):

```
<nav aria-label="Primary navigation" class="usa-nav" role="navigation">
```

Instead, we want something like

```
<nav th:aria-label="#{navigation.primaryNavigation}" class="usa-nav" role="navigation">
```

# Notes
* [Internationalization guide](https://github.com/civiform/civiform/wiki/Internationalization-%28i18n%29)

# Done when

- [ ] All `aria-label`s in CiviForm's .html files use `th:aria-label` instead of `aria-label`

**Comment by ssandbekkhaug** [comment]
@erikang20 Here's another starter issue related to internationalization and accessibility

**Comment by sijiayam** [comment]
@Erikang20 @melanie-exygy What would be the best way to review and test this?

**Comment by shreyachatterjee00** [comment]
@dkatzz Plaban to verify

**Comment by melanie-exygy** [comment]
I suggest that we verify that all element accessible names are translated, rather than limiting ourselves to aria-labels.

**Comment by sijiayam** [comment]
@melanie-exygy @dkatzz I could definitely QA and check all translations. Are there specific pages I should avoid testing (ex. the applicant homepage and program overview page?)

**Comment by melanie-exygy** [comment]
> [@melanie-exygy](https://github.com/melanie-exygy) [@dkatzz](https://github.com/dkatzz) I could definitely QA and check all translations. Are there specific pages I should avoid testing (ex. the applicant homepage and program overview page?)

Hey @sijiayam, I thought Plaban was going to do the testing.  Anyhow, program overview is ready for this type of testing, but probably not the applicant homepage.

**Comment by plabanTest1** [comment]
Verified the aria-labels for all the translations.
There are 2 languages from dropdown whose translations doesn't seem to work. Screenshot below.

<img width="1792" alt="Image" src="https://github.com/user-attachments/assets/f234e15f-4365-48ae-92f5-59a054c4f1dc" />
